src/domain/ledger/index.types.d.ts

```diff
@@ -80,23 +81,23 @@ type OnChainTxArgs = TxArgs & {
 
 type AddLnTxReceiveArgs = LnTxArgs & {
   currency: WalletCurrency
-  fiat: FiatAmount | undefined
-  usdFeeInboundLiquidity: FiatAmount
+  usd: UsdAmount | undefined
```

Comment:
Isn't this only needed for USD wallets?
Callers of this function already know (via the WalletCurrency) wether or not we are receiving in USD or BTC - so we should also segregate those call paths to the ledger - also because there is different logic (dealer entries) on the ledger side.

ie. instead of `use: UsdAmount | undefined` there should be 
`AddLnTxReceiveArgs` and `AddLnTxReceiveInUsdArgs` or something like that.

Reply:
should be done in another PR when we introduce the dealer?